src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-filter/TranscriptsFilter.scss

```diff
@@ -1,9 +1,9 @@
 @import 'src/styles/common';
-@import '../default-transcripts-list/constants';
+@import '../../constants';
```

Comment:
This one needs changing as well. Otherwise, error:

![image](https://user-images.githubusercontent.com/6834224/92220326-c15e3000-ee93-11ea-9ebb-be1c4df8db54.png)

Reply:
Yes, this was missed. Added now